docker-compose.yaml

```diff
@@ -4,12 +4,13 @@
 
 services:
   redis:
-    image: "redis/redis-stack"
+    image: "dockerhub.nvidia.com/redis/redis-stack"
```

Comment:
I was running into rate limits against dockerhub.io on internal machines, this mirror helped me get away from it.  As this merge is into performance branch, i thought this is fine, let me know if it is not ?